### ISSUE TITLE
Crashed Car model

### ISSUE THREAD

**Issue by alimalkhalifa** [xackery/mp#10]
We need a crashed car model. Maybe a 20s or 30s renault.

For the intro into the forest

**Comment by xackery** [comment]
[2:45 PM] Almighty Ra!: Ideally we could stick in the car in some brush and just have smoke coming out or something
[2:46 PM] Almighty Ra!: Though I guess it would be nice to have a car interior perspective
[2:46 PM] Vitoria: just need to deform the front a bit
[2:46 PM] Vitoria: have the hood be popped from the deformation
[2:46 PM] Xackery: we can cut corners for both
[2:46 PM] Xackery: but yeah a 2-stage car possibly
[2:46 PM] Almighty Ra!: Maybe also a SUPER SIMPLE road, with trees on both sides

**Comment by ghost** [comment]
![1280px-Citroen_trefle.jpg](https://user-images.githubusercontent.com/68444036/87990079-925a4d80-cae3-11ea-9e94-7ab7daa09f23.jpg)

Citroën Type C 5HP Torpedo (1922-1926)

**Comment by xackery** [comment]
this is part of https://github.com/xackery/mp/issues/2

**Comment by xackery** [comment]
Ian made a prototype of this, untextured, needs to get uploaded

**Comment by xackery** [comment]
asked Ian about uploading it. WAiting on reply. I believe texturing isn't done either.
